### PR TITLE
Update Installation Guide for Ubuntu 14.04 as well

### PR DESCRIPTION
**Pull request by kithokit**
This installation guide is not updated for ubu14.04.

Slightly change for it so that other users can benefit

### CODE REVIEW

File: doc/install/installation.md [hunk 1]
```diff
@@ -31,11 +31,19 @@ the default editor.
...
-    sudo apt-get install redis-server 
+    sudo apt-get install redis-server
 
 # 2. Ruby
+Download Ruby and compile it (Ubuntu 14.04):
 
-Download Ruby and compile it:
+    mkdir /tmp/ruby && cd /tmp/ruby
+    curl --progress http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p481.tar.bz2 | tar xj
+    cd ruby-2.0.0-p481
```
Comment (vsizov) [inline doc/install/installation.md]: Can you explain what is the reason of changing ruby version for newer ubuntu?

**Comment by kithokit** [comment]
Hi, it is becoz ruby cannot be compiled using this version on 14.04.

A more stable version of 2.0.0 is suggested on 14.04

http://stackoverflow.com/questions/26801289/problems-compiling-ruby

File: doc/install/installation.md [hunk 2]
```diff
@@ -74,13 +82,16 @@ You can use either MySQL or PostgreSQL.
...
     # Logout MYSQL
     mysql> exit;
-    
+
 ### PostgreSQL
 
-    # Install the database packages
+    # Install the database packages (Ubuntu 14.04)
+    sudo apt-get install -y postgresql-9.3 libpq-dev
+
```
Comment (vsizov) [inline doc/install/installation.md]: What's the difference here?

**Comment by jvanbaarsen** [comment]
This merge request has been closed because a request for more information has not been reacted to for more than 2 weeks. If you respond and conform to the merge request guidelines in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#pull-requests) we will reopen this merge request.

**Pull request closed without merging**